test/fixtures/templates/preserve-ie-conditional-comments/test.js

```diff
@@ -0,0 +1,3 @@
+exports.templateData = {
```

Comment:
How is the test cases generated and run ? I don't see assertions for the expected.html and template.marko generated.  I think there is a someother suite runs it